client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -741,12 +738,12 @@ class PatientSubmissionScreen extends React.Component {
     return true;
   }
 
-  handlePractitionerOptionSelected(value) {
+  handlePractitionerOptionSelected(license) {
     const { actions } = this.props;
     const { practitionerOptions } = this.state;
-    const option = practitionerOptions.find(o => stringifyPractionerOption(o) === value);
+    const option = practitionerOptions.find(o => o.license === license);
 
-    if (option) {
+    if (option != null) {
```

Comment:
Here we explicitely want to check for not null?  No need to check for empty, undefined, NaN, false or 0?

Reply:
`option` is the result of the `find` function, which returns `undefined` if the value is not found. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) So there is no need to check for NaN, flase, 0 etc.
I use the `!= null` operator (notice the double equality), which include `null` and `undefined` ...